src/aat/resources/features/F-040/F-040_Later_Case_Read_By_Solicitor_3.td.json

```diff
@@ -0,0 +1,166 @@
+          "AddressLine2": null,
+          "AddressLine3": null
+        },
+        "DocumentField6": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/76976634-374f-472b-a819-a3a938b4cc83",
+          "document_filename": "Screenshot 2019-11-13 at 11.25.45 am.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/76976634-374f-472b-a819-a3a938b4cc83/binary"
+        },
+        "DocumentField1": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/e636eef3-839a-48cd-96f0-dad43e66f5c4",
+          "document_filename": "Screenshot 2019-11-13 at 3.24.26 pm.png",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/e636eef3-839a-48cd-96f0-dad43e66f5c4/binary"
+        },
+        "ComplexField": {
+          "ComplexTextField": null,
+          "ComplexNestedField": {
+            "NestedNumberField": null,
+            "NestedCollectionTextField": []
+          },
+          "ComplexFixedListField": null
+        },
+        "FixedRadioListField": "RADIOVALUE2",
+        "DateTimeField": null,
+        "PhoneUKField": null,
+        "NumberField": "111111222222",
+        "MultiSelectListField": [
+          "OPTION5",
+          "OPTION4",
+          "OPTION3"
+        ],
+        "YesOrNoField": "Yes",
+        "EmailField": "testemail@gmail.com",
+        "TextField": "sometextfield",
+        "DateField": "2222-01-02",
+        "TextAreaField": "textarea",
+        "CollectionField": []
+      },
+      "data_classification": {
+        "DocumentField4": "PUBLIC",
+        "MoneyGBPField": "PUBLIC",
+        "DocumentField3": "PUBLIC",
+        "FixedListField": "PUBLIC",
+        "AddressUKField": {
+          "value": {
+            "County": "PUBLIC",
+            "Country": "PUBLIC",
+            "PostCode": "PUBLIC",
+            "PostTown": "PUBLIC",
+            "AddressLine1": "PUBLIC",
+            "AddressLine2": "PUBLIC",
+            "AddressLine3": "PUBLIC"
+          },
+          "classification": "PUBLIC"
+        },
+        "DocumentField6": "PUBLIC",
+        "DocumentField1": "PUBLIC",
+        "FixedRadioListField": "PUBLIC",
+        "DateTimeField": "PUBLIC",
+        "PhoneUKField": "PUBLIC",
+        "NumberField": "PUBLIC",
+        "MultiSelectListField": "PUBLIC",
+        "YesOrNoField": "PUBLIC",
+        "EmailField": "PUBLIC",
+        "TextField": "PUBLIC",
+        "DateField": "PUBLIC",
+        "TextAreaField": "PUBLIC",
+        "CollectionField": {
+          "value": [],
+          "classification": "PUBLIC"
+        }
+      },
+      "after_submit_callback_response": null,
+      "callback_response_status_code": null,
+      "callback_response_status": null,
+      "delete_draft_response_status_code": null,
+      "delete_draft_response_status": null,
+      "security_classifications": {
+        "DocumentField4": "PUBLIC",
```

Comment:
Can use inheritance by creating a common file. Please check the feasibility.

Reply:
Siva, it is being used at only one place. So we can skip the inheritance for now.